test/testdata/cfg/rescue_bad_class.rb.flatten-tree.exp

```diff
@@ -1,5 +1,5 @@
 begin
-  <emptyTree>
+  ::<Magic>.<define-top-class-or-module>(::<root>)
```

Comment:
My only question is whether it's ok for this send to exist outside of a method at this point (i'm not sure it'll get walked to by the CFG).

Reply:
Great question.
AFAIK it isn't fine and it will not be walked by the CFG

Reply:
The intent was for this call to end up in static-init. Maybe the code to construct static-init needs to be moved to after when the classes are moved 🤔

Reply:
Should be fixed by moving https://github.com/sorbet/sorbet/blob/de19ef0cbf4d1ade80d522398ece1c0bfdb0034b/class_flatten/class_flatten.cc#L68 to postTransform

Reply:
@elliottt I'm still not seeing it in static init?

Reply:
Yes, this is `::root`, which is OK to skip. This is what I was talking about in https://github.com/sorbet/sorbet/pull/2829#discussion_r402505532